tox.ini

```diff
@@ -33,5 +33,13 @@ commands =
 
 extras = dev
 
+# The current version of python-olm that's on PyPI does not include a types marker.
+# Ideally we'd just specify a custom pip install command that uses an --index-url that
+# points to Olm's PyPI repo, but it looks like the packaging does not include py.typed in
+# wheels. https://gitlab.matrix.org/matrix-org/olm/-/merge_requests/62 is an attempt at
+# fixing this.
+deps =
+  git+https://gitlab.matrix.org/babolivier/olm.git@babolivier/py.typed_manifest#egg=python-olm&subdirectory=python
+
```

Comment:
it would be good to switch to olm's main repo, once https://gitlab.matrix.org/matrix-org/olm/-/merge_requests/62 is merged

Reply:
The plan once that MR is merged is to replace this block with:

```ini
install_command = python -m pip install --index-url=https://gitlab.matrix.org/api/v4/projects/27/packages/pypi/simple {opts} {packages}
```

This will cause pip to fetch all packages from the gitlab-hosted pypi repo for olm. If a package cannot be found there (which will be the case for any package that isn't `python-olm`) GitLab will automatically forward the request to pypi.org according to https://gitlab.matrix.org/help/user/packages/pypi_repository/index#install-a-pypi-package (as of GitLab 14.2, and I've confirmed with ops that gitlab.matrix.org runs a more recent version of GitLab).

Longer term, the hope is to have a package on pypi.org (either for just the bindings, or the bindings + a build version of olm), but that's currently blocked on legal concerns which I'm told will take at least a month to resolve.

Reply:
Olm 3.2.13 has been released with the change, so I've now put that plan in motion.